apps/mobile/src/components/browser/browser-sheet/tabs/browser-connected-tab.tsx

```diff
@@ -0,0 +1,79 @@
+import { Dimensions } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { useApps } from '@/store/apps/apps.read';
+import { t } from '@lingui/macro';
+import { Image } from 'expo-image';
+
+import { Box, Text } from '@leather.io/ui/native';
+
+import { ScreenshotCard } from '../screenshot-card';
+import { URL_SEARCH_HEIGHT } from '../utils';
+import { BrowserTabLayout } from './browser-tab.layout';
+
+const { width } = Dimensions.get('window');
+
+interface ConnectedProps {
+  goToUrl(url: string): void;
+}
+
+export function BrowserConnectedTab({ goToUrl }: ConnectedProps) {
+  const { list: appList } = useApps('connected');
+  const { bottom } = useSafeAreaInsets();
+
+  if (appList.length === 0) {
+    return (
+      <Box
+        flex={1}
+        justifyContent="center"
+        alignItems="center"
+        alignSelf="center"
+        maxWidth={width * 0.5}
+        style={{
+          paddingBottom: bottom + URL_SEARCH_HEIGHT,
+        }}
+        gap="3"
+      >
+        <Image
+          style={{ height: width * 0.5, width: width * 0.5 }}
+          contentFit="cover"
+          source={require('@/assets/connected_empty.png')}
```

Comment:
Can we import this a different way than an inline `require`? 

Maybe this should live in the UI package?